sections/conclusion.tex

```diff
@@ -1,2 +1,36 @@
 \chapter{Conclusion}\label{ch:conclusion}
-There are probably still some bugs in the theme. If you should find one, then please submit it on \url{https://github.com/jkjaer/aauLatexTemplates}.
+The code that we overtook from last year for the search engine was incomplete and did not work.
+We therefore implemented the database from scratch using C\# and the Entity Framework Core.
```

Comment:
We didn't implement a database. We set up a Postgres database with redesigned schemas and implemented an API to inferface with it.